test/check-application

```diff
@@ -673,6 +673,11 @@ class TestApplication(testlib.MachineCase):
             b.set_input_text("#run-image-dialog-memory-limit input.form-control", "0.5")
             b.set_val('#memory-unit-select', "GiB")
 
+            b.set_checked("#run-image-dialog-cpu-priority-checkbox", True)
+            b.wait_present("#run-image-dialog-cpu-priority-checkbox:checked")
+            b.wait_present('div.modal-body label:contains("CPU Priority") + div.form-inline > input[value="1024"]')
+            b.set_input_text("#run-image-dialog-cpu-priority input.form-control", "512")
+
```

Comment:
Could you please make this a separate test (not necessarily new function, just a new container), and also cover the default setting? We should test both.